backendless.d.ts

```diff
@@ -1049,8 +1049,14 @@ declare module __Backendless {
         loginSync(userName: string, password: string, stayLoggedIn?: boolean): Backendless.User;
         loginSync<T>(userName: string, password: string, stayLoggedIn?: boolean): T;
 
-        login(idenity: string, password: string, stayLoggedIn?: boolean): Promise<Backendless.User>;
-        login<T>(idenity: string, password: string, stayLoggedIn?: boolean): Promise<T>;
+        login(identity: string, password: string, stayLoggedIn?: boolean): Promise<Backendless.User>;
+        login<T>(identity: string, password: string, stayLoggedIn?: boolean): Promise<T>;
+
+        loginAsGuestSync(stayLoggedIn?: boolean): Backendless.User;
```

Comment:
perhaps, we should not implement `Sync` methods, because all the sync methods will be removed in JS-SDK@6.x

Reply:
I know, but added this for consistency. All synchronous methods anyway will be deleted, including this one, and I don’t think that leaving it now will cause some difficulties.

Reply:
ok, agree